src/Request/Credentials/Response.php

```diff
@@ -11,22 +14,23 @@ class Response extends BaseRequest
     protected $accountId;
 
     /**
-     * Setups the final data
-     * @return $this
+     * Current account id
+     *
+     * @return int|null
      */
-    protected function setupData()
+    public function accountId()
```

Comment:
není ještě náhodou rule na doplnění typů?

```php
$rectorConfig->rules([
        ReturnTypeFromStrictNativeCallRector::class,
        ReturnTypeFromStrictScalarReturnExprRector::class,
    ]);
```

Reply:
Rector na to rules ma. Udelal bych to az budu prechazet na level 6 phpstanu